bcipy/acquisition/tests/test_client.py

```diff
@@ -75,7 +75,10 @@ def test_daq_with_no_buffer(self):
         daq.start_acquisition()
         time.sleep(0.1)
         daq.stop_acquisition()
-
+        
+        #Make sure we are able to stop the buffer process
+        buf_temp = daq._buf
```

Comment:
I'm confused why this is needed to stop the buffer process? @lawhead @smektala

Reply:
When the buffer server is started, daq._buf is set to a reference to the queue that is used to communicate with the server. This takes place when the daq object is initialized. However, for the test to work correctly, daq._buf has to be set to None. When daq.cleanup() is run, it tries to send a stop message to the server process, but if the variable that should contain the message queue reference is still set to None, it isn't able to stop the buffer server process, and so it runs forever in the background, causing Pytest to hang when all tests are finished. The change saves the queue reference in buf_temp, so that daq._buf can be set back to the queue reference once the test is finished, and the buffer server process can be stopped.